addons/beehave/nodes/beehave_tree.gd

```diff
@@ -22,6 +22,8 @@ enum {
 
 var actor : Node
 
+var _tree_metric_name : String
```

Comment:
we may want to give this a better name: as it is tracking the `process_time` of this tree, perhaps something like this:
```suggestion
var _process_time_metric_name : String
```
(we are inside the tree already so using "tree" is redundant)